code/ui/tttpanel/TTTPanel.cs

```diff
@@ -27,27 +27,26 @@ public void AddChild(Panel child, int index)
         {
             AddChild(child);
 
-            int childCount = ChildCount;
-
-            if (childCount > 1)
-            {
-                if (index < 0 || index >= childCount)
-                {
-                    return;
-                }
-
-                Panel tmp = _children[index];
-                _children[index++] = this;
-
-                while (index < childCount)
-                {
-                    _children[index] = tmp;
-                    tmp = _children[++index];
-                }
-
-                _children[index] = tmp;
-                _children[++index] = null;
-            }
+            // int childCount = ChildCount;
```

Comment:
Do we need to keep this commented out?
Could we just remove it?

Reply:
I need this for upcoming child repositioning, currently, it keeps crashing the game, that's why it's commented out